### PR TITLE
Add documentation for serverless cdk plugin

### PR DESCRIPTION
**Pull request by GuillaumeLagrange**
- chore(cdk-plugin): add variable resolver type
- feat(cdk-plugin): add helper to resolve variables in cdk output
- feat(cdk-plugin): resolve variables in cdk cloudformation output
- chore: add serverless-cdk-plugin to README.md
- docs: add how-to for serverless cdk plugin

### CODE REVIEW

File: backend/orchestrator/resources/dynamodb.ts [hunk 1]
```diff
@@ -14,7 +14,9 @@ export class OrchestratorDynamodb extends ServerlessCdkPlugin.ServerlessConstruc
   constructor(scope: Construct, id: string, serverlessProps: ServerlessProps) {
     super(scope, id, serverlessProps);
 
-    const table = new Table(this, 'OrchestratorTable', {
+    const table = new Table(this, `OrchestratorTable`, {
+      // tableName: `table-${serverlessProps.serverless.provider.stage ?? ''}`,
+      tableName: `table-${serverlessProps.service.provider.stage}`,
```
Comment (fargito) [inline backend/orchestrator/resources/dynamodb.ts:19]: Setting the resource names is not a good practive IMO, do you think you could demontrate this capability with another setting?

Comment (fargito) [reply backend/orchestrator/resources/dynamodb.ts:19]: For example, a worthy demonstration would be to set the a different deletion policy depending on the stage, WDYT?

Comment (GuillaumeLagrange) [reply backend/orchestrator/resources/dynamodb.ts]: Yes this was just to test things out, and I could not get the actions to run with the PR set as draft. I gave an example in the docs now :)

Comment (GuillaumeLagrange) [reply backend/orchestrator/resources/dynamodb.ts]: I included your excellent idea in the example as well, thanks !

**Review by fargito — APPROVED** [review]

**Comment by fargito** [comment]
@GuillaumeLagrange I just realized you did not open this PR against `main`, but against one of your branches, was this intentional?

**Pull request merged**